src/lib/selectors.ts

```diff
@@ -57,7 +57,7 @@ import { SendableMessageType } from '../utils';
 /**
  * const sdk = selectors.getSdk(state);
  */
-export const getSdk = (state: SendBirdState): SendbirdGroupChat & SendbirdOpenChat => {
+export const getSdk = (state: SendBirdState) => {
```

Comment:
Why remove return type?

Reply:
`SendBirdState` already has the type specified for the SDK.
During the returning it, the function's return type is correctly inferred

Reply:
ok but personally i think its better to keep return type for all functions for consistency and readability. But this is minor.

Reply:
Yes, it can vary based on personal preferences.
I personally don't prefer redundant type declarations for inferred types.
I tend to specify them only where it's necessary, as it can make the code harder to read when it gets longer.

If a function has multiple branches and complex logic with many possible return types, I do specify the return type for such cases to prevent errors